charts/kube-prometheus-stack/values.yaml

```diff
@@ -2538,6 +2538,12 @@ prometheusOperator:
   #   cpu: 100m
   #   memory: 100Mi
 
+  ## Operator Environment
+  ##  env:
+  ##    VARIABLE: value
+  env:
+    GOGC: "30"
```

Comment:
Why are you setting it to 30 when 100 is the default ?

Reply:
I read the PR you linked, makes a lot of sense now :D

Reply:
```
The latter. My observation is that the memory usage increases a lot initially when the operator populates its local cache but it's transient because most of the data is discarded by the transform func and reclaiming more aggressively than the default is beneficial in this case.
```
From the discussion between @philipgough  and @simonpasquier. ref: https://github.com/prometheus-operator/prometheus-operator/pull/5993/files/2c67ca6fc45549a588bd39d362b9bad167b0c025#r1393911325

Reply:
Thank you for the contribution @dongjiang1989 :)